WooCommerce/Classes/Authentication/Jetpack Setup/Native Jetpack Setup/LoginJetpackSetupViewModel.swift

```diff
@@ -0,0 +1,81 @@
+import Foundation
+import Yosemite
+
+/// View model for `LoginJetpackSetupView`.
+///
+final class LoginJetpackSetupViewModel: ObservableObject {
```

Comment:
I guess we will be adding unit tests for this class in a future PR.

Reply:
Yes I'll add the tests later when we have more logic in this view model.